src/buildApplication/getVercelConfig.ts

```diff
@@ -28,6 +28,57 @@ export async function getVercelConfig(): Promise<VercelConfig> {
 	return config;
 }
 
+/**
+ * Gets the routes of a specific vercel phase
+ *
+ * @param config Vercel config object containing all the routes information
+ * @param phase the phase from which extract the routes
+ * @returns the phases' routes
+ */
+export function getPhaseRoutes(
+	config: VercelConfig,
+	phase: VercelPhase
+): VercelRoute[] {
+	const routes: VercelRoute[] = [];
+
+	if (!config.routes) {
+		return [];
+	}
+
+	let currentRouteIdx = 0;
+
+	if (phase === 'none') {
+		while (
+			config.routes[currentRouteIdx] &&
+			!(config.routes[currentRouteIdx] as VercelHandler).handle
+		) {
+			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Assigning the item in the array to a const variable and checking the truthiness of that variable would avoid these non-null assertions.

Reply:
To remove the eslint comment I thought of two alternatives:
![Screenshot 2023-06-06 at 12 58 07](https://github.com/cloudflare/next-on-pages/assets/61631103/feb15989-39e0-4ffa-9e54-5aea700d7826)

any preference? (or a different idea?)

(PS: unfortunately both to me feel less clear than the current code 😓)